### PR TITLE
Update selinux context for postgres volume

### PR DESCRIPTION
**Pull request by stephenpc**
Potential fix for https://github.com/ansible/awx/issues/128

### CODE REVIEW

File: installer/local_docker/tasks/main.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@
     state: started
     image: postgres:9.6
     volumes:
-      - "{{ postgres_data_dir }}:/var/lib/postgresql/data"
+      - "{{ postgres_data_dir }}:/var/lib/postgresql/data:Z"
```
Comment (matburt) [inline installer/local_docker/tasks/main.yml:104]: does this work generally on all platforms?

Comment (stephenpc) [reply installer/local_docker/tasks/main.yml:104]: @matburt I've tested successfully on **CentOS 7.x** and **Ubuntu 16.04 LTS**. I think these are fairly representative of RHEL based and Debian based distros.

**Comment by Zokormazo** [comment]
By the way, postgres recommends to use another directory inside the /var/lib/postgresql/data when using filesystem mountpoints

https://hub.docker.com/_/postgres/ -> ENV variables,->  PGDATA
> This optional environment variable can be used to define another location - like a subdirectory - for the database files. The default is /var/lib/postgresql/data, but if the data volume you're using is a fs mountpoint (like with GCE persistent disks), Postgres initdb recommends a subdirectory (for example /var/lib/postgresql/data/pgdata ) be created to contain the data.

So it could be added to this PR or add another one for it.

**Comment by matburt** [comment]
That's out of scope for this PR and the original issue. I'll try this one out on Mac and Arch Linux here in a little bit and we'll merge it if all looks good.

**Review by matburt — APPROVED** [review]
Looks good everywhere I've tested it.

**Pull request merged**